### ISSUE TITLE
Input group size small

### ISSUE THREAD

**Issue by Guria** [elementalui/elemental#59]
``` jsx
<InputGroup contiguous>
    <InputGroup.Section grow>
        <FormInput type="text" placeholder="Input group field" size="sm" />
    </InputGroup.Section>
    <InputGroup.Section>
        <Button size="sm">Button</Button>
    </InputGroup.Section>
</InputGroup>
```

![](http://storage8.static.itmages.com/i/15/0918/h_1442565539_1381285_08f9bb8b05.png)

**Comment by jossmac** [comment]
@Guria thanks for the heads up

**Comment by NogsMPLS** [comment]
What browser/OS is this in?

Latest Chrome, Firefox, and IE10+ on Windows 7 machine looks good to me. (although contiguous does seem to work correctly on IE10)

**Comment by jossmac** [comment]
@Guria having trouble reproducing this, would be great to get your system/browser details. Please note that Elemental supports:
- Chrome (mobile and desktop)
- Safari (mobile and desktop)
- Firefox
- Internet Explorer 10 and up

**Comment by jossmac** [comment]
Resolved by 8e05148
